go.sum

```diff
@@ -0,0 +1,29 @@
+github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
```

Comment:
You might want to run `go mod tidy`, looks like you have some ref to non existant deps here.

Reply:
I believe it's indirect from the rest-layer and mgo dependencies. Have attempted to both run `go mod tidy`, and to delete the `go.sum` and run `go mod tidy` again. The result is no changes to this file

Reply:
Unrelated, I can perhaps remove the testify dependency from `mongo_test.go` as well before merge. It will make the `testify` dependency indirect, so that when it's no longer used in `rest-layer`, it would go away here as well.